google_checkstyle.xml

```diff
@@ -1,6 +1,11 @@
 <?xml version="1.0"?>
 <!DOCTYPE module PUBLIC "-//Puppy Crawl//DTD Check Configuration 1.3//EN" "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
 
+<!-- 
+    Adjusted from original Google Checkstyle config to match requirements for swagger-codegen
+    See detailed comments next to changes
```

Comment:
I shall add a link to the original file.  However, I shan't be paying a great deal of attention to comments on grammar in a *comment* on a *configuration file* where it's patently obvious what's going on.